packages/wdio-junit-reporter/src/types.ts

```diff
@@ -16,6 +16,12 @@ interface ClassNameFormatOptions {
     suite?: SuiteStats,
 }
 
+interface TestSuiteNameFormatOptions {
+    name?: string,
+    activeFeatureName?: any
+    suite?: SuiteStats
```

Comment:
I suggest we find some more consistency here. Do we need two parameters? Can we maybe settle on one name for it?